server/src-lib/Hasura/GraphQL/Transport/HTTP.hs

```diff
@@ -84,64 +120,112 @@ runGQ
   -> GQLReqUnparsed
   -> m (HttpResponse EncJSON)
 runGQ env logger reqId userInfo ipAddress reqHeaders queryType reqUnparsed = do
-  -- The response and misc telemetry data:
-  let telemTransport = Telem.HTTP
-  (telemTimeTot_DT, (telemCacheHit, telemLocality, (telemTimeIO_DT, telemQueryType, !resp))) <- withElapsedTime $ do
+  (telemTimeTot_DT, (telemQueryType, telemTimeIO_DT, telemLocality, resp)) <- withElapsedTime $ do
     E.ExecutionCtx _ sqlGenCtx pgExecCtx {- planCache -} sc scVer httpManager enableAL <- ask
 
     -- run system authorization on the GraphQL API
     reqParsed <- E.checkGQLExecution userInfo (reqHeaders, ipAddress) enableAL sc reqUnparsed
                  >>= flip onLeft throwError
 
-    (telemCacheHit, execPlan) <- E.getResolvedExecPlan env logger pgExecCtx {- planCache -}
+    (_telemCacheHit, execPlan) <- E.getResolvedExecPlan env logger pgExecCtx {- planCache -}
```

Comment:
Is it okay that we’re throwing this information away? And if so, can it be eliminated from the result altogether?

Reply:
Well, since we're not doing query plan caching at the moment, its value will always be `Telem.Miss`. But I've now pushed two commits that plumb this constant value through the code :)